examples/example-fastify-web-mcd/README.md

```
@@ -0,0 +1,80 @@
+
+## Install dependencies
+
+```bash
+npm install
+```
+
+## Add `/etc/hosts` entries
+
+So that both hostnames resolve to your machine, add the following to `/etc/hosts`:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)